### DISCUSSION TITLE
Deux répertoires de vues avec setTemplateDirRoot

### DISCUSSION THREAD

**Discussion by moimp3** [rawsrc/PhpEcho#28] [Q&A] [answered]
Bonjour,
Est-il possible d'avoir deux répertoires de vues avec setTemplateDirRoot (par exemple un pour l'usage général et un autre pour le répertoire admin)

**Comment by rawsrc** [comment, marked as answer]
Salut,

Non c'est impossible.
D'ailleurs je ne vois pas trop l'intérêt.

Quand tu identifies un administrateur, tu charges un fichier de config pour
les administrateurs dans lequel tu définies la racine des vues.
Et tu fais de même pour ceux qui ne sont pas administrateurs.

rawsrc
